hermes_core/util/tests/test_schema.py

```diff
@@ -306,24 +306,24 @@ def test_min_max_TT2000():
     """Get min/max values for TT2000 types"""
     minval, maxval = HermesDataSchema()._get_minmax(const.CDF_TIME_TT2000)
     # Make sure the minimum isn't just plain invalid
-    assert minval == datetime.datetime(1, 1, 1)
-    assert maxval == datetime.datetime(2292, 4, 11, 11, 46, 7, 670776)
+    assert minval == datetime.datetime(1900, 1, 1, 0, 0, 0, 0)
```

Comment:
Shouldn't we be using astropy time objects here? since we are supposed to be using them everywhere?

Reply:
Spacepy loads these attributes as `datetime` objects when loading a CDF file. So if we want to make everything astropy time everywhere, we would need to create wrappers around the spacepy attribute loading as well as update these derivation functions. If we want to go that route that might warrant a separate issue and PR.

Reply:
Grrrr. spacepy does support astropy time but does not use it "natively".